### PR TITLE
unique learner id for each multilabel wrapper

### PR DESCRIPTION
**Pull request by QuayAu**
This is a PR for #1700

### CODE REVIEW

**Comment by larskotthoff** [comment]
What exactly does your test do? Notice that not all equal isn't the same as all not equal.

**Comment by QuayAu** [comment]
I wanted to check, if the ids are not all equal. But apparently this doesn't do that.
I will have a look at it later.

File: R/MultilabelClassifierChainsWrapper.R [hunk 1]
```diff
@@ -23,7 +23,7 @@
 #' @example inst/examples/MultilabelWrapper.R
 makeMultilabelClassifierChainsWrapper = function(learner, order = NULL) {
   learner = checkLearner(learner, type = "classif", props = "twoclass")
-  id = paste("multilabel", learner$id, sep = ".")
+  id = paste("multilabel.cc", learner$id, sep = ".")
```
Comment (giuseppec) [inline R/MultilabelClassifierChainsWrapper.R]: use `stri_paste`  everywhere

File: R/MultilabelBinaryRelevanceWrapper.R [hunk 2]
```diff
@@ -27,7 +27,7 @@
 #' @example inst/examples/MultilabelWrapper.R
 makeMultilabelBinaryRelevanceWrapper = function(learner) {
   learner = checkLearner(learner, type = "classif")
-  id = stri_paste("multilabel", learner$id, sep = ".")
+  id = stri_paste("multilabel.br", learner$id, sep = ".")
```
Comment (giuseppec) [inline R/MultilabelBinaryRelevanceWrapper.R]: We should use the getter `getLearnerId`  here.

File: R/MultilabelBinaryRelevanceWrapper.R [hunk 3]
```diff
@@ -27,7 +27,7 @@
...
   packs = learner$package
   x = makeHomogeneousEnsemble(id, learner$type, learner, packs,
```
Comment (giuseppec) [inline R/MultilabelBinaryRelevanceWrapper.R]: getLearnerType

File: R/MultilabelClassifierChainsWrapper.R [hunk 4]
```diff
@@ -23,7 +23,7 @@
...
   packs = learner$package
```
Comment (giuseppec) [inline R/MultilabelClassifierChainsWrapper.R]: getLearnerPackages

File: tests/testthat/test_base_multilabel.R [hunk 5]
```diff
@@ -61,6 +61,18 @@ test_that("MultilabelBinaryRelevanceWrapper with glmnet", {
   expect_error(pred, NA)
 })
 
+test_that("check unique id", {
+  lrn = makeLearner("classif.rpart")
+  id1 = getLearnerId(makeMultilabelBinaryRelevanceWrapper(lrn))
```
Comment (giuseppec) [inline tests/testthat/test_base_multilabel.R]: you could also directly add a test to check if the id contaisn the expected string, e.g. `expect_string(id1, pattern = "multilabel.br.classif.rpart")`

**Comment by berndbischl** [comment]
@QuayAu 
please fix what was said quickly so we can merge

**Comment by larskotthoff** [comment]
ping @QuayAu

**Comment by pat-s** [comment]
@QuayAu 	The initial issue is closed - can this PR also be closed?

**Pull request closed without merging**